tools/clang/include/clang/Basic/Version.h

```diff
@@ -18,6 +18,7 @@
 
 #include "clang/Basic/Version.inc"
 #include "llvm/ADT/StringRef.h"
+#include <cstdint>
```

Comment:
Nothing in this header should need cstdint so this seems like an odd addition here.

Reply:
Without this  Im getting:
 
```
In file included from /home/marcin/vulkan/vulkansdk-linux-x86_64-1.3.243.0/1.3.243.0/source/DirectXShaderCompiler/tools/clang/lib/Basic/Version.cpp:14:
/home/marcin/vulkan/vulkansdk-linux-x86_64-1.3.243.0/1.3.243.0/source/DirectXShaderCompiler/tools/clang/include/clang/Basic/Version.h:85:3: error: ‘uint32_t’ does not name a type
   85 |   uint32_t getGitCommitCount();
      |   ^~~~~~~~
/home/marcin/vulkan/vulkansdk-linux-x86_64-1.3.243.0/1.3.243.0/source/DirectXShaderCompiler/tools/clang/include/clang/Basic/Version.h:21:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   20 | #include "llvm/ADT/StringRef.h"
  +++ |+#include <cstdint>
```

Reply:
Doh! I don't know how I missed that. You're right 😄 .

Reply:
Can you add `// HLSL Change - for uint32_t.` to this line so that we know that the include was added for DXC and why?

Reply:
Done